certified-connectors/Payables E2E/apiDefinition.swagger.json

```diff
@@ -0,0 +1,132 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "IN-D Invoice Data Capture",
```

Comment:
@gayathri35 is this title correct? The PR is named `IN-D Payables E2E` and the folder's name is `Payables E2E`.